be/src/column/binary_column.h

```diff
@@ -157,7 +157,15 @@ class BinaryColumn final : public ColumnFactory<Column, BinaryColumn> {
 
     bool append_continuous_strings(const std::vector<Slice>& strs) override;
 
-    size_t append_numbers(const void* buff, size_t length) override { return -1; }
+    // BinaryColumn::append_numbers allows 1 number to be appended every time.
+    // The type of the column value is Slice, whose size is not fixed.
+    // Hence, we cannot determine the number of numbers appended.
+    size_t append_numbers(const void* buff, size_t length) override {
```

Comment:
It seems that this method is Equivalent to `BinaryColumn::append(const Slice&)`

Reply:
The implement of `append_numbers` is removed from the `BinaryColumn`.